.pre-commit-config.yaml

```diff
@@ -1,6 +1,6 @@
 repos:
```

Comment:
```suggestion
ci:
  autofix_prs: true
  autoupdate_commit_msg: "[pre-commit.ci] pre-commit suggestions"
  autoupdate_schedule: quarterly
  # submodules: true

repos:
```

Reply:
`autofix_prs: true` is the default and can be omitted. 

I usually don't change the commit message either, but no strong opinion on that.

`autoupdate_schedule: quarterly` is good, much less noisy than weekly!

Remove `# submodules: true` if not used.

Reply:
> `autofix_prs: true` is the default and can be omitted.

True, but I think it is better to repeat it so everyone is aware of it :)